docs/docs/guides/GETTING_STARTED.mdx

```diff
@@ -100,6 +100,9 @@ You'll likely import 3D related files when using react-native-filament. To make
 
 ```js
 const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+const defaultAssetExts = require("metro-config/src/defaults/defaults").assetExts;
```

Comment:
I think expo uses a different import for this? So maybe its better to take the value from the default config?

Reply:
I'm not sure how Expo handles it, I've made the update to take value from default config,